tests/integration/integration/home.spec.js

```diff
@@ -0,0 +1,17 @@
+import { Selector } from "testcafe";
```

Comment:
The path of this file is `tests/integration/integration/...`

Should we keep the structure like the unit test structure?
`tests/integration/helpers/....js`
`tests/integration/pages/login.spec.js`
`tests/integration/pages/home.spec.js` etc?